data/transit/route/bus.json

```diff
@@ -1086,6 +1086,19 @@
         "route": "bus"
       }
     },
+    {
+      "displayName": "Berrys Coaches",
+      "locationSet": {"include": ["gb"]},
+      "tags": {
+        "network": "Berrys Coaches",
+        "network:wikidata": "Q28121432",
+        "network:wikipedia": "Berry's Coaches",
```

Comment:
Don't forget the `en:` on the wiki tags 😄 
```suggestion
        "network:wikipedia": "en:Berry's Coaches",
```

Reply:
Oh no, such a rookie mistake :)

Reply:
> Oh no, such a rookie mistake :)

haha I laughed when I saw it - you're definitely not a rookie :)